Remotion/ObjectBinding/Web/UI/Controls/IBocVisibleColumnDefinition.cs

```diff
@@ -0,0 +1,12 @@
+﻿// SPDX-FileCopyrightText: (c) RUBICON IT GmbH, www.rubicon.eu
+// SPDX-License-Identifier: LGPL-2.1-or-later
+namespace Remotion.ObjectBinding.Web.UI.Controls
+{
+  /// <summary>
+  /// Interface to handle visibility for <see cref="BocColumnDefinition"/>s.
+  /// </summary>
+  public interface IBocVisibleColumnDefinition
```

Comment:
As discussed, the `IsVisible` flag should please be added to `BocColumnDefinition` since its usable with all types of columns. We shouldn't need the extra interface because when pushing the API to `BocColumnDefinition`.